### PR TITLE
Add links to  playground for several fixes

### PR DESCRIPTION
**Pull request by TobiasNx**
Add links in docu to playground examples for several fixes. More examples can be added later.

### CODE REVIEW

**Comment by TobiasNx** [comment]
@blackwinter you can merge since I am away for four month starting tomorrow.

File: README.md [hunk 1]
```diff
@@ -178,6 +180,8 @@ Defines an external map for [lookup](#lookup) from a file or a URL. Maps with mo
 put_filemap("<sourceFile>", "<mapName>", sep_char: "\t")
 ```
 
+[Playground-Example](https://metafacture.org/playground/?example=put_filemap)
```
Comment (blackwinter) [inline README.md:183]: Link doesn't work; see metafacture/metafacture-playground#153:

```suggestion
[Playground-Example](https://metafacture.org/playground/?example=put+filemap)
```

File: README.md [hunk 2]
```diff
@@ -833,6 +902,8 @@ end
 call_macro("<macroName>"[, <dynamicLocalVariables>...])
 ```
 
+[Playground-Example](https://metafacture.org/playground/?example=do+pu+macro)
```
Comment (blackwinter) [inline README.md:905]: Typo:

```suggestion
[Playground-Example](https://metafacture.org/playground/?example=do+put+macro)
```

**Review by blackwinter — CHANGES_REQUESTED** [review]
I would prefer to wait for metafacture/metafacture-playground#153 to be resolved. I'll update the links accordingly.

Any objections?

**Pull request merged**